pkg/identityserver/storetest/user_store.go

```diff
@@ -373,6 +373,58 @@ func (st *StoreTest) TestUserStoreCRUD(t *T) {
 		// 	a.So(errors.IsNotFound(err), should.BeTrue)
 		// }
 	})
+
+	t.Run("CreateAfterPurge", func(t *T) {
+		a, ctx := test.New(t)
+		var err error
+		start := time.Now().Truncate(time.Second)
+		stamp := start.Add(-1 * time.Minute)
+
+		// UserId have to be the same as the one deleted in the previous test case.
+		created, err = s.CreateUser(ctx, &ttnpb.User{
```

Comment:
We already test `Create` above, so feel free to provide only `Ids` and `PrimaryEmailAddress` to have a minimal test.

I also think we should do `Create` -> `Delete` -> `Restore` -> `Get` -> `Delete` -> `Purge`, in a loop.